clients/admin-ui/src/features/common/form/inputs.tsx

```diff
@@ -434,6 +447,7 @@ export const CustomSelect = ({
                 isDisabled={isDisabled}
                 singleValueBlock={singleValueBlock}
                 menuPosition={props.menuPosition}
+                onChange={!isFormikOnChange ? onChange : undefined}
```

Comment:
The snippet below (lines 193-195) was added for the custom field form modal in order to dynamically update the validationSchema. 
```typescript
    if (onChange) {
      onChange(newValue);
    }
```


`isFormikOnChange` on change was added because the `CustomSelect` field in `CustomFieldList` is using formik field props which adds an onChange. That onChange function errors out. To work around this I added this boolean to only pass in the onChange if it's one that manually passed in by us. 


If this causes issues in the future we can probably add a more robust fix

Reply:
I agree. I'll create a follow up ticket for it